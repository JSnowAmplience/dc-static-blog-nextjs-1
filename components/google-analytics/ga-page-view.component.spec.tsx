import renderer from 'react-test-renderer';
import GaPageView from './ga-page-view.component';

describe('GaPageView', (): void => {
  it('should render some google analytics capable of tracking page views', (): void => {
    const wrapper = renderer.create(<GaPageView />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
