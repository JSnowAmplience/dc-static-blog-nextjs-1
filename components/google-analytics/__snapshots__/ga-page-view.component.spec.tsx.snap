// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GaPageView should render some google analytics capable of tracking page views 1`] = `
Array [
  <script
    async={true}
    src="https://www.google-analytics.com/analytics.js"
  />,
  <script
    dangerouslySetInnerHTML={
      Object {
        "__html": "window.ga=window.ga||function(){(ga.q = ga.q || []).push(arguments)};ga.l=+new Date; ga('create', 'undefined', 'auto'); ga('send', 'pageview');",
      }
    }
  />,
]
`;
